src/encoder.rs

```diff
@@ -1868,6 +1867,33 @@ fn encode_partition_topdown(fi: &FrameInvariants, fs: &mut FrameState,
     }
 }
 
+use rayon::prelude::*;
+
+
+#[inline(always)]
+fn build_coarse_pmvs(fi: &FrameInvariants, fs: &FrameState) -> Vec<[Option<MotionVector>; REF_FRAMES]> {
+// initial coarse ME loop
```

Comment:
Strange indentation here.
```suggestion
  // initial coarse ME loop
```

Reply:
I need to do a rustfmt pass indeed!